src/components/KDropdown/KDropdownItem.vue

```diff
@@ -143,7 +143,7 @@ const availableComponents = computed((): DropdownItemRenderedRecord => ({
   link: {
     tag: 'a',
     attrs: {
-      class: `dropdown-item-trigger ${props.disabled ? 'disabled' : ''} ${attrs.class || ''}`,
+      class: 'dropdown-item-trigger',
```

Comment:
suggestion: since this class is reused across each element, can we set it to a `const` above and reuse it to ensure it doesn't get inadvertently changed in a future PR?

Reply:
Done